### PR TITLE
Metadata: Mozilla.sccache version 0.3.1

### PR DESCRIPTION
**Pull request by emabrey**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

WIP, see mozilla/sccache#1251

Currently waiting for access to pre-release version of winget for ability to test with `zipInstall` enabled

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90637)

### CODE REVIEW

**Comment by emabrey** [comment]
@microsoft-github-policy-service agree

File: manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: zip
+InstallModes:
+- silent
+UpgradeBehavior: uninstallPrevious
+Commands:
+- sccache
+Installers:
+- Architecture: x64
+  InstallerType: zip
```
Comment (Trenly) [inline manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml:16]: Zip installers must have the nested installer type and nested installer files set

Comment (emabrey) [reply manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml:16]: The 1.2.0 JSON schema doesn't support those keys though. 1.4.0 support I couldn't get working locally, though it does support those keys. I have an orphaned commit with my attempt at doing that, I'll comment with the commit hash when I'm back at my computer, so maybe you can tell me what I'm missing.

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/m/Mozilla/sccache/0.3.1/Mozilla.sccache.installer.yaml:16]: > The 1.2.0 JSON schema doesn't support those keys though. 1.4.0 support I couldn't get working locally, though it does support those keys. I have an orphaned commit with my attempt at doing that, I'll comment with the commit hash when I'm back at my computer (so maybe you can tell me what I'm missing).

1.4 is officially supported; I suggest using `YamlCreate.ps1 Mozilla.sccache 0.3.1 2` which will re-prompt for the installer URL, walk you through the nested files, and update all the manifests to v1.4

**Comment by Anutrix** [comment]
Can someone salvage this?

**Pull request closed without merging**